storage/src/main/java/org/apache/uniffle/storage/common/AbstractStorage.java

```diff
@@ -44,13 +47,13 @@ public ShuffleWriteHandler getOrCreateWriteHandler(CreateShuffleWriteHandlerRequ
     writerHandlers.computeIfAbsent(request.getAppId(), key -> JavaUtils.newConcurrentMap());
     requests.computeIfAbsent(request.getAppId(), key -> JavaUtils.newConcurrentMap());
     Map<String, ShuffleWriteHandler> map = writerHandlers.get(request.getAppId());
-    String partitionKey =
-        RssUtils.generatePartitionKey(
-            request.getAppId(), request.getShuffleId(), request.getStartPartition());
-    map.computeIfAbsent(partitionKey, key -> newWriteHandler(request));
+    String partitionKeyExceptAppId =
```

Comment:
Shall we forward a new step to use `int` type and bit `shift` operation to construct a int type partitionKey?

This can reduce string concat operation and the invoked time of `valueOf`, so it can reduce jvm objects and save the cpu usage.

Reply:
Ok for me